explorer/src/database.ts

```diff
@@ -26,7 +27,7 @@ const loadOptions = (env?: string) => {
 // Loads the following ENV vars, giving them precedence.
 // i.e. TYPEORM_PORT will replace "port" in ormconfig.json.
 const mergeOptions = (): PostgresConnectionOptions => {
-  const envOptions: { [key: string]: string } = {}
+  const envOptions: { [key: string]: any } = { logger: new TypeOrmLogger() }
```

Comment:
This isn't really an `envOption` is it? I would probably just specify the logger on line 37 directly inline, then there's no need to mess with the typing of `envOptions`